src/schema.ts

```diff
@@ -239,5 +242,17 @@ export const comment = {
   },
 };
 
+export const html: Schema = {
+  attributes: {
+    content: { type: String, required: true },
+  },
+
+  transform(node) {
+    return node.attributes.content
+      ? new Raw(node.attributes.content, node.inline)
+      : null;
```

Comment:
Do we need this ternary?

Reply:
Eh, we probably don't. I don't think there's a case where the parser actually returns any empty html token, so this may have been overly defensive.